go/libraries/doltcore/sqle/enginetest/dolt_queries_diff.go

```diff
@@ -3677,6 +3677,96 @@ var UnscopedDiffSystemTableScriptTests = []queries.ScriptTest{
 	},
 }
 
+var ColumnDiffSystemTableScriptTests = []queries.ScriptTest{
```

Comment:
Would be great to add more tests on cases such as 'add column', 'drop column', 'modify column', etc. I like to use the existing tests on similar features as template to track different test cases I am not aware of and some edge cases as well 😁